src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
 
+### CA3075: Insecure DTD processing in XML ###
+
+Using XmlTextReader.Load(), creating an insecure XmlReaderSettings instance when invoking XmlReader.Create(), setting the InnerXml property of the XmlDocument and enabling DTD processing using XmlUrlResolver insecurely can lead to information disclosure. Replace it with a call to the Load() method overload that takes an XmlReader instance, use XmlReader.Create() to accept XmlReaderSettings arguments or consider explicitly setting secure values. The DataViewSettingCollectionString property of DataViewManager should always be assigned from a trusted source, the DtdProcessing property should be set to false, and the XmlResolver property should be changed to XmlSecureResolver or null.�
+
```

Comment:
I think this will be confusing (frankly, for the same reason I think the composite rule is confusing). The first sentence lists four things you might do wrong. Then the second sentence provides the fixes, but not in a one-to-one correspondence with the problems. Then the third sentence mentions more problems.

Are these descriptions meant to be a single, unformatted paragraph of text? (@michaelcfanning). If not, I might write something like this:

> Certain coding practices related to XML DTDs are insecure and can lead to information disclosure:
* When calling an overload of `XmlReader.Create` that accepts an `XmlReaderSettings` argument, ensure that the `DtdProcessing` property of the settings object is set to a secure value such as `Ignore` or `Prohibit`.
* Do not set the `InnerXml` property of an `XmlDocument` object.
* _etc_

If it does have to be a single unformatted paragraph, we can collapse it like this:

> Certain coding practices related to XML DTDs are insecure and can lead to information disclosure. When calling an overload of `XmlReader.Create` that accepts an `XmlReaderSettings` argument, ensure that the `DtdProcessing` property of the settings object is set to a secure value such as `Ignore` or `Prohibit`. Do not set the `InnerXml` property of an `XmlDocument` object. _etc_

This groups each problem together with its solution.